hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HddsServerUtil.java

```diff
@@ -511,6 +512,18 @@ public static SCMSecurityProtocol getScmSecurityClient(
         SCMSecurityProtocol.class, conf);
   }
 
+  public static SecretKeyProtocolScm getScmSecretClient(
```

Comment:
Add a javadoc to mention it's intended to be used by clients under SCM identify. Maybe rename it to `getSecretKeyClientForSCM` to make the intention clear.